content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -282,51 +331,49 @@ It'll analyze the changes, use existing results from the <abbr>run cache</abbr>,
 and execute only the commands needed to produce new results (model, metrics,
 plots).
 
-The same logic applies to other possible adjustments — edit source code, update
-datasets — you do the changes, use `dvc repro`, and DVC runs what needs to be
-run.
+The same workflow applies to other modifications; you make changes (editing
+source code, configs files and updating datasets), and then you simply (re)run
+`dvc repro`. DVC runs whatever needs running.
 
-## Comparing iterations
```

Comment:
I think here it still makes sense, why removing it? (Trails, at least for now are considered independent)

Reply:
See [this comment](https://github.com/iterative/dvc.org/pull/4508#pullrequestreview-1409502623) for context.
It makes sense here as well as there, but I agree that this is a bit too advanced and "heavy" to get into here and more associated with using pipelines for ML experimentation.

It still [mentioned](https://github.com/iterative/dvc.org/pull/4519/files#diff-81064b0c5efb920d52d22aa2584913c9c2f8411c49917141b7e9e0850cc47f37R347) btw, not completely removed
Please take a look at the review app to get the "impact change"

Reply:
I saw it. It's still very hard for me to justify this kind of changes and it goes against a bit of the original idea that I tried to explain

This section is still about ML, not data eng, software eng, etc, etc. Those plots, metrics are very very much ML-specific. I would argue if we decide to keep trail independent we should move it to `dvc exp` here, show the table, etc, etc.

Reply:
I'm not sure I follow exactly what you have in mind, but I don't see how bringing `dvc exp` in here makes sense in the current structure. I understand how "everything is about ML" and that everything can be tied together, but we're trying to take the complexity and amount of functionality and make it digestable in small portions.
You can even say that extensive explanation of more advanced commands don't belong in get-started at all (it's only get-started, it can't be 100% extensive because than it will be all of the docs!)

This being about ML doesn't change. this is about ML->data-management->metrics,params,plots. And I do think it's a very good feedback we got here, that there is more than enough content without going intro comparison operations between runs.